ocs_ci/ocs/ui/views.py

```
@@ -224,7 +224,8 @@
         By.CSS_SELECTOR,
     ),
     "storage_system_tab": (
-        'a[data-test-id="horizontal-link-Storage System"]',
+        'a[data-test-id="horizontal-link-Storage System"],'
+        'a[data-test="nav"][href*="storage-cluster"]',
```

Ruff (0.16.1)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)